src/components/CurrencyInputPanel/index.tsx

```diff
@@ -329,27 +332,17 @@ export default function CurrencyInputPanel({
               >
                 <Aligner>
                   <RowFixed>
-                    {hideLogo ? null : pair ? (
-                      <DoubleCurrencyLogo currency0={pair.token0} currency1={pair.token1} size={20} margin={true} />
-                    ) : currency ? (
-                      <CurrencyLogo currency={currency || undefined} size={'20px'} />
-                    ) : null}
-                    {pair ? (
-                      <StyledTokenName className="pair-name-container">
-                        {pair?.token0.symbol}:{pair?.token1.symbol}
-                      </StyledTokenName>
-                    ) : (
-                      <StyledTokenName
-                        className="token-symbol-container"
-                        active={Boolean(currency && currency.symbol)}
-                        fontSize={fontSize}
-                        style={{ paddingRight: disableCurrencySelect ? '8px' : 0 }}
-                      >
-                        {(nativeCurrency?.symbol && maxCurrencySymbolLength
-                          ? shortString(nativeCurrency.symbol, maxCurrencySymbolLength)
-                          : nativeCurrency?.symbol) || <Trans>Select a token</Trans>}
-                      </StyledTokenName>
-                    )}
+                    {currency && !hideLogo ? <CurrencyLogo currency={currency} size={'20px'} /> : null}
+                    <StyledTokenName
+                      className="token-symbol-container"
```

Comment:
What's this classname for?

Reply:
I don't know, old code.
I think for QA automation test